cmd/activator/main.go

```diff
@@ -175,7 +175,7 @@ func main() {
 	throttler := activatornet.NewThrottler(ctx, env.PodIP)
 	go throttler.Run(ctx)
 
-	oct := tracing.NewOpenCensusTracer(tracing.WithExporter(networking.ActivatorServiceName, logger))
+	oct := tracing.NewOpenCensusTracer(tracing.WithExporterFull(env.PodName, env.PodIP+":8012", logger))
```

Comment:
is `8012` correct here? Isn't that the port used for HTTP1 traffic? HTTP2 would go via `8013`.